test/unit/test_data_utils.py

```diff
@@ -252,3 +253,12 @@ def test_check_data_redundancy_throws_exception_pipe(self):
         pb_file_paths = ['pb_files']
         with self.assertRaises(Exception):
             data_utils.check_data_redundancy(pb_file_paths[0], pb_file_paths[1])
+            
+    def test_pyarrow_to_parquet_conversion_does_not_throw_exception(self):
+        try:
+             df = pd.DataFrame({'x': [1,2]})
+             df.to_parquet('test.parquet', engine='pyarrow')
+        except Exception as e:
+            assert False, "conversion to parquet should not fail"
```

Comment:
```suggestion
    def test_pyarrow_to_parquet_conversion_regression_issue_106(self):
        df = pd.DataFrame({'x': [1,2]})
        df.to_parquet('test.parquet', engine='pyarrow')
```

We don't want the exception to be swallowed if it does fail.

Reply:
If you want to add an assertion, you check check that the values are equivalent before and after conversion.

Reply:
On the exception it will fail the test, thus will fail the build as well.

Reply:
Yes, but surfacing the actual exception will make the failure obvious and debugging easier instead of swallowing it with a generic "assert False".